config/rewrites.d/support.rackspace.com.json

```diff
@@ -436,5 +436,12 @@
         "rewrite": false,
         "status": 301
       }
```

Comment:
so should there be one there and then on line 444 that I added? I just copied the previous block.